src/ui/docengine.cpp

```diff
@@ -5,8 +5,11 @@
 #include <QTextCodec>
 #include <QTextStream>
 #include <QCoreApplication>
+#include <QPushButton>
```

Comment:
?

Reply:
Necessary because `msgBox.addButton(..)` returns a `QPushButton*`. Wasn't needed with the previous `setStandardButtons(..)` but using custom buttons means using `QPushButtons`/`QAbstractButtons` directly.